smartcontractkit/external-adapters-js
# Add implied price adapter

## Changes

- Added implied price EA



## Steps to Test

1. Run integration tests
2. (Or) run the composite EA with source adapter(s) (see README)
3. Send a request to EA (see integration tests or test-payload.json)

## Quality Assurance

- [x] If a new adapter was made, or an existing one was modified so that its environment variables have changed, update the relevant `infra-k8s` configuration file. https://github.com/smartcontractkit/infra-k8s/pull/2449
